frontend/src/components/pages/Scheduling/DonationInformation.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Container } from "@chakra-ui/react";
+import React from "react";
+import { NavigationProps, SetForm } from "react-hooks-helper";
+import { useHistory } from "react-router-dom";
+
+import { SchedulingFormProps } from "./types";
+
+const DonationInformation = ({
+  formValues,
+  setForm,
+  navigation,
+}: {
+  navigation: NavigationProps;
+  formValues: SchedulingFormProps;
+  setForm: SetForm;
+}) => {
+  const { next } = navigation;
+  const history = useHistory();
+  //   const { insert form fields for this page here } = formData;
+
+  return (
+    // Insert Select Date and Time page here
+    <Container>
+      This is Donation Information page
+      <Button
+        mt="2"
+        backgroundColor="black.100"
+        color="white.100"
+        size="md"
+        w="100%"
+        onClick={next}
+      >
```

Comment:
Since this is the same button on all the pages, we should probably move it to a separate [variant](https://chakra-ui.com/docs/theming/component-style) to import a bit similar to [here](https://github.com/uwblueprint/planet-read/blob/main/frontend/src/theme/components/Button.ts):

```typescript
const Button = {
//base styles
variants: {
    nextStep: {
        marginTop: "2rem",
        backgroundColor: "black.100",
        color: "white.100",
        width: "100%"
    }
  }
}

```